packages/titus-backend/Dockerfile

```diff
@@ -1,15 +1,11 @@
-FROM node:16-alpine as builder
-RUN apk add dumb-init
+FROM node:16.13.1-alpine3.14 as builder
```

Comment:
as an exception to my earlier statement, unless we have an automated mechanism to keep this up to date I wouldn't want us to hardcode a specific version. this is because we are usually ok to use a major version here. node 16, whatever is the latest. I would revert it to the earlier image name. I would still put a dependency mechanism in place as per #1467

Reply:
Pros and cons already discussed in #1467. As a result, the generic tag is going to be used.